sktime/transformations/panel/rocket/_minirocket.py

```diff
@@ -85,14 +95,14 @@ class MiniRocket(BaseTransformer):
 
     def __init__(
         self,
-        num_kernels=10_000,
+        num_kernels=9_996,
```

Comment:
we need to leave it at 10.000 as otherwise it may break user code that uses `get_params`.